src/App.jsx

```diff
@@ -14,16 +14,24 @@ function App() {
   return (
     <>
       <HashRouter>
-        <Header />
-        <Routes>
-          <Route path="/" element={<Home />} />
-          <Route path="/staff" element={<Staff />} />
-          <Route path="/contact" element={<Contact />} />
-          <Route path="/sharing-code" element={<SharingCode />} />
-          <Route path="/getting-help" element={<GettingHelp />} />
-          <Route path="/faq" element={<FAQ />} />
-        </Routes>
-        <Footer />
+        <div className="columns is-flex-direction-column page-wrapper">
+          <div className="column is-narrow">
```

Comment:
Removing is-narrow from footer and header and then viewing on a tall screen causes header, main-content, and footer all to be centered. This can be checked by zooming out to 50% in browser. Leaving is-narrow for header and footer keeps them at the top/bottom.